### PR TITLE
ESLint: Enable more styling rules

### PR DESCRIPTION
**Pull request by Turbo87**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -11,47 +11,89 @@ module.exports = {
...
     // JSHint "expr"
     'no-unused-expressions': [2, {
       allowShortCircuit: true,
       allowTernary: true,
     }],
 
+    // JSHint "immed", disabled already in .jshintrc
+    'wrap-iife': 0,
+
+    'yoda': 2,
```
Comment (stefanpenner) [inline .eslintrc.js]: whats this one?

**Comment by stefanpenner** [comment]
LGTM

File: .eslintrc.js (hunk 1 above)
Comment (Turbo87) [inline .eslintrc.js]: for example: 

```js
if ("red" === color) {
    // ...
}
```

see http://eslint.org/docs/rules/yoda

**Comment by stefanpenner** [comment]
@homu r+

**Pull request merged**